pkg/aws/install.go

```diff
@@ -312,13 +312,13 @@ func InstallCluster(name, releaseImage, dhParamsFile string, waitForReady bool)
 	}
 	log.Infof("Created VPN load balancer with ARN: %s and DNS: %s", vpnLBARN, vpnLBDNS)
 
-	vpnTGARN, err := aws.EnsureUDPTargetGroup(lbInfo.VPC, vpnLBName, vpnNodePort, apiNodePort)
+	vpnTGARN, err := aws.EnsureTargetGroup(lbInfo.VPC, vpnLBName, vpnNodePort)
 	if err != nil {
 		return fmt.Errorf("cannot create VPN target group: %v", err)
 	}
 	log.Infof("Created VPN target group ARN: %s", vpnTGARN)
 
-	if err = aws.EnsureTarget(vpnTGARN, machineID); err != nil {
+	if err = aws.EnsureTarget(vpnTGARN, machineIP); err != nil {
```

Comment:
can you explain why the change from machineID to machineIP?

Reply:
AWS UDP load balancers require instance targets instead of IP targets. https://aws.amazon.com/blogs/aws/new-udp-load-balancing-for-network-load-balancer/
> Supported Targets – UDP on Network Load Balancers is supported for Instance target types (IP target types and PrivateLink are not currently supported).

IP targets have less security requirements because they don't require that instances are exposed to the outside world through the security group.